src/vivarium_census_prl_synth_pop/components/person_migration.py

```diff
@@ -226,16 +226,19 @@ def _perform_non_reference_person_moves(
         to_move = movers
         household_id_values = pop.loc[movers, "household_id"]
 
-        for iteration in range(10):
-            if to_move.empty:
-                break
-
-            household_id_values[to_move] = self.randomness.choice(
-                to_move,
-                choices=households_with_non_movers,
-                additional_key=f"non_reference_person_move_household_ids_{iteration}",
-            )
-            to_move = movers[household_id_values == pop.loc[movers, "household_id"]]
+        # There is an edge case in very small populations where there are no households
+        # to move to; in that case, the move is not performed
```

Comment:
Does this require a doc update or is it edgy enough?

Reply:
Good question! I can make a doc update for this. It is _super_ edgy so I think this can be an "implement first, document later" situation.